src/reducers/forms.js

```diff
@@ -6,6 +6,12 @@ export const defaultState = {
 
 const formState = (state = defaultState, action) => {
   switch (action.type) {
+    case 'CAPTURE_FIELD_INPUT': {
+      const { id, value } = action;
+      const newState = state;
+      newState.values[id] = value;
+      return newState;
```

Comment:
Excellent.. This can also be written as: 
```
      const { id, value } = action;
      const newState = state;
      
      return {
        ...newState,
        [id]: value,
      }
```